frontends/ol-components/src/components/LearningResourceExpanded/InfoSection.tsx

```diff
@@ -86,6 +87,10 @@ const Certificate = styled.div`
     height: 16px;
   }
 `
+// The SignupPopover needs to display over the top of the resource drawer
+const StyledSignupPopover = styled(SignupPopover)({
+  zIndex: 9999,
```

Comment:
it would be good to avoid the hardcoded zIndex if at all possible (realizing this might be the only workaround for now)